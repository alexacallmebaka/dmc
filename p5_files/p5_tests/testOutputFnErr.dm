a : () int {

return 5;

}

main : () void {

give a;

}
