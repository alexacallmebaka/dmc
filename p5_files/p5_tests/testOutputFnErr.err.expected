FATAL [8,8]-[8,9]: Attempt to output a class
FATAL [9,8]-[9,9]: Attempt to output a function
Type Analysis Failed
